revoked.md

```diff
@@ -0,0 +1,9 @@
+# Revoked Behaviors and Methods
+The Malware Behavior Catalog has revoked a small number of behaviors and methods as it has evolved.  The IDs for revoked behaviors and methods are not re-used.  If it appears that we are missing behavior or methods IDs, this page will explain what revoked IDs previously represented, when they were retired, and the reason for retiring them.
+
+| Revoked ID | Behavior or Method | Revocation Date | Reason |
+|--- |--- |--- |--- |
+| E1608 | Install Certificate | Feb. 14, 2024 | Changed from enhanced ATT&CK technique ID (E-prefix) to enhanced ATT&CK sub-technique ID (F-prefix).  Current ID is F0016 |
```

Comment:
please use the same date format as we use for behavior versioning: e.g., 14 February 2024